js/siteorigin-panels/view/builder.js

```diff
@@ -549,13 +563,23 @@ module.exports = Backbone.View.extend( {
         // Make sure we actually need to copy content.
         if( panelsOptions.copy_content && this.attachedToEditor && this.$el.is(':visible')) {
 
+			var postId;
+			if(this.builderType === this.builderTypes.EDITOR_ATTACHED) {
+				postId = $('#post_ID').val();
+			} else if(this.builderType === this.builderTypes.CUSTOM_HOME_PAGE) {
+				postId = $('#panels-home-page').data('postId');
+			} else {
+				// This shouldn't happen.
```

Comment:
This actually can happen. I'm not sure if the `builderType` I added is the same as the one you added, but the reason I originally did this was to pass the builder type along with the request for style fields.

The builder type can be set by anything that wants to use a builder view. For example the Post Type Builder sets the `data-type` field [here](https://github.com/siteorigin/so-cpt-builder/blob/develop/tpl/admin-build.php#L5), which is used by Page Builder [here](https://github.com/siteorigin/siteorigin-panels/blob/develop/js/siteorigin-panels/main.js#L92).

Just checking that this PR takes this into account.